### DISCUSSION TITLE
Privacy Manifect in IOS App

### DISCUSSION THREAD

**Discussion by mohammadprgrm20123** [codemagic-ci-cd/codemagic-docs#2720] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

![837b09cb-9abf-455d-9a34-687fbe33dac4](https://github.com/user-attachments/assets/bb60cd35-7211-4dbf-8464-a903c4ada2ad)

### Expected results

![837b09cb-9abf-455d-9a34-687fbe33dac4](https://github.com/user-attachments/assets/ce28186c-c74a-4887-96b3-9698a4d9c01c)

how do i do ?

How to create a Privacy Manifect in My ios App In Code Magic

please help me 

### Actual results

...

**Comment by mohammadprgrm20123** [comment]
I don't have MacOs

**Reply by dtrdic** [reply]
hello @mohammadprgrm20123 please check this [official Apple documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files) on how to create a privacy manifest.

Since you don't have a MacOS machine locally, then you would need to access the build machine and create it, then just copy it from the build machine to your local machine and add it to your app.

So in order to access the build machine:
1. Click on enable SSH/VNC checkbox
﻿​
![image](https://github.com/user-attachments/assets/570172f2-bd8b-4989-9af7-4cc259b3490d)




﻿2. the Explore build machine via SSH or VNC client button will appear
﻿​
![image](https://github.com/user-attachments/assets/316068eb-bd08-4f2c-8d35-5a918dcebc70)



Then just enter the host address, username, and password into VNC viewer

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
